manifests/a/aluxnimm/outlookcaldavsynchronizer/4.5.0/aluxnimm.outlookcaldavsynchronizer.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: aluxnimm.outlookcaldavsynchronizer
+PackageVersion: 4.5.0
+PackageLocale: en-US
+Publisher: Gerhard Zehetbauer
+PublisherUrl: https://caldavsynchronizer.org/
+PublisherSupportUrl: https://caldavsynchronizer.org/help/issues/
+PrivacyUrl: https://caldavsynchronizer.org/de/datenschutzerklaerung/
+PackageName: CalDavSynchronizer
+PackageUrl: https://caldavsynchronizer.org/
+License: AGPL-3.0 license
+Copyright: COPYRIGHT © 2023 BY Generalize-IT Solutions OG
+ShortDescription: Sync Outlook Everywhere
+Description: Outlook CalDav Synchronizer is a free Outlook Plugin, which synchronizes events, tasks and contacts between Outlook and Google, SOGo, Nextcloud or any other CalDAV or CardDAV server. Supported Outlook versions are Office 365, 2019, 2016, 2013, 2010 and 2007.
+Moniker: outlookcaldavsynchronizer
+Tags:
+- caldav
+- csharp
+- opensource
+- outlook
+- plugin
+- sync-outlook

```

```suggestion
- sync-outlook
ReleaseNotes: |
  Released 2024/12/26
  New features
    - Update NuGet packages.
    - Use Unicolour library for colour comparison and remove ColorMine package.
    - Use json site to query newest version.
```

From https://github.com/aluxnimm/outlookcaldavsynchronizer/releases/tag/v4.5.0